fastfood/food.py

```diff
@@ -42,33 +41,23 @@ def build_cookbook(build_config, templatepack, cookbooks_home,
     with open(build_config) as cfg:
         cfg = json.load(cfg)
 
+    cookbook_name = cfg['name']
     cookbook = None
-    if cookbook_path:
-        try:
-            cookbook = book.CookBook(cookbook_path)
```

Comment:
Removing these lines, ~46-52, does that mean you can't run `build` on an existing cookbook? Now the `cookbook` variable is always being set to `None` and so the `if not cookbook` condition below won't be needed. I am pretty sure this makes sense but is this what we want?